python/ray/_private/gcs_utils.py

```diff
@@ -142,6 +142,9 @@ def cleanup_redis_storage(
     if not isinstance(storage_namespace, str):
         raise ValueError("storage namespace must be a string")
 
-    # Right now, GCS store all data into a hash set key by storage_namespace.
-    # So we only need to delete the specific key to cleanup the cluster.
-    return del_key_from_storage(host, port, password, use_ssl, storage_namespace)
+    # Right now, GCS stores all data into multiple hashes with keys prefixed by
+    # storage_namespace. So we only need to delete the specific key to cleanup the
+    # cluster.
+    return del_external_storage_namespace_from_storage(
```

Comment:
I feel the old name is cleaner: `del_key_prefix_from_storage`. Or `del_keys_with_prefix_from_storage` is even better IMO.

Reply:
I wanted to hide the key schema (that external_storage_namespace is prefix of the keys). But ok I can change that back to prefix.

Reply:
Got it. Then I think we don't have to make the name generic, it can be just `cleanup_redis_storage()`. I'm merging for now and we can rename it later.